pom.xml

```diff
@@ -36,7 +39,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>structs</artifactId>
-            <version>1.9</version>
+            <version>1.20</version>
```

Comment:
use the jenkins bom instead of choosing all these versions directly

https://github.com/jenkinsci/bom

you could introduce it in a separate PR to reduce the diff here as well